www-apache/modsecurity-crs/modsecurity-crs-3.3.4.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+DESCRIPTION="OWASP ModSecurity Core Rule Set"
+HOMEPAGE="https://coreruleset.org/"
+SRC_URI="https://github.com/coreruleset/coreruleset/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64 x86"
+
+RDEPEND="|| ( >=www-apache/mod_security-2.9.6 >=dev-libs/modsecurity-3.0.8 )"
+
+S="${WORKDIR}/coreruleset-${PV}"
+
+src_install() {
+	insinto "/usr/share/${PN}"
+	doins -r rules util
+
+	dodoc CHANGES CONTRIBUTING.md CONTRIBUTORS.md KNOWN_BUGS README.md crs-setup.conf.example
+
+	if has_version www-servers/apache; then
+		# I don't think it's worth pulling in apache-module.eclass just for
+		# this path...
+		insinto /etc/apache2/modules.d
+		doins "${FILESDIR}/80_mod_security-crs.conf"
+	fi
+
+	if has_version www-servers/nginx; then
+		insinto "/etc/modsecurity"
+		doins -r crs-setup.conf.example
```

Comment:
I'm not sure we need this example in /etc/modsecurity since we copy it to /usr/share/docs and it probably needs to be changed for each system anyway. The `-r` flag is also unnecessary as it's a single file.

Reply:
Removed.